gstools/krige/base.py

```diff
@@ -16,7 +16,7 @@
 from scipy.spatial.distance import cdist
 import scipy.linalg as spl
 from gstools.field.base import Field
-from gstools.krige.krigesum import krigesum
+from gstools.krige.krigesum import krigesum, krigesumfast
```

Comment:
`krigesum_fast`?

Reply:
Renamed to `calc_field_krige_and_variance` and `calc_field_krige`.